backend/routes/assignments.js

```diff
@@ -147,4 +148,86 @@ router.get('/grades/:assignmentId', auth, async (req, res) => {
 
 })
 
+/**
+ * Fetch all grades for given assignment
+ * Route: GET
+ * Params: assignmentId
+ */
+router.get('/grades/all/:assignmentId', auth, async (req, res) => {
+    const assignmentId = req.params.assignmentId;
+    try {
+        try {
+            error.str(assignmentId);
+        } catch (e) {
+            return res.status(400).json({ error: e.message });
+        }
+        const response = await userData.fetchAllGrades(assignmentId);
+        res.status(200).json({ "status": "success", "grades": response });
```

Comment:
Assuming change is made to fetchAllGrades to return { "status": "success", "grades": grades }, this can just be the same:
res.status(200).json(response);
as the other routes

Reply:
I have made changes to just return response, this will be reflecting in latest PR.